copier/questions/package_details.yml

```diff
@@ -1,36 +1,28 @@
 ---
 # package details
-AddPackageDetails:
-  type: bool
-  default: yes
-  help: |-
-    If you are planning to publish the package it is recommended to provide more details.
-       Do you want to provide extra details for the package?
-  when: "{{ template_profile != 'minimum'}}"
+# AddPackageDetails:
+#   type: bool
+#   default: yes
+#   help: |-
+#     If you are planning to publish the package it is recommended to provide more details.
+#        Do you want to provide extra details for the package?
+#   when: false
 package_short_description:
   type: str
   placeholder: Short description
   help: Short description of package
-  when: "{{ template_profile != 'minimum' and AddPackageDetails }}"
   validator: >-
         {% if not package_short_description %}
         Package description cannot be empty
         {% endif %}
-keyword1:
+keywords:
   type: str
-  placeholder: keyword1
-  help: Add keyword1
-  when: "{{ template_profile != 'minimum' and AddPackageDetails }}"
-keyword2:
-  type: str
-  placeholder: keyword2
-  help: Add keyword2
-  when: "{{ template_profile != 'minimum' and AddPackageDetails }}"
+  placeholder: keyword1,second keyword,third keyword
+  help: Add keywords to make your package findable on PyPI
```

Comment:
Do we want to add a validator e.g.
 validator: >-
        {% if not keywords %}
        Please provide at least one keyword
        {% endif %}

Reply:
I created an issue about this.